src/physicsml/lightning/graph_datasets/neighbourhood_list_torch.py

```diff
@@ -63,11 +63,18 @@ def construct_edge_indices_and_attrs(
     self_interaction: bool,
 ) -> Tuple[torch.Tensor, Optional[torch.Tensor], torch.Tensor]:
     if (pbc is not None) and (cell is not None):
+        cell = cell.type(positions.dtype).to(positions.device)
+
+        # solve pos = coefs^T @ cell
+        # pos = pos - torch.floor(coefs) @ cell
+        coefs = torch.floor(torch.matmul(positions, torch.inverse(cell)))
+        wrapped_positions = positions - torch.matmul(coefs, cell)
```

Comment:
that's pretty neat!

Reply:
I think this is the source of the bug I reported in #35.  It causes positions to get wrapped on all axes, even if the elements of `pbc` are `False`.